hydra/test_utils/launcher_common_tests.py

```diff
@@ -495,16 +556,23 @@ def test_custom_sweeper_run_workdir(
         task_config = OmegaConf.create(task_config or {})  # type: ignore
         cfg = OmegaConf.merge(task_launcher_cfg, task_config)
         assert isinstance(cfg, DictConfig)
+        expected_outputs = (
+            str(self.get_test_app_working_dir() / expected_dir)  # type: ignore
```

Comment:
why the type ignore?

Reply:
mypy complains since ```self.get_test_app_working_dir()``` returns an ```Optional[Path]```, but we check in the if else so it should be fine.


```
hydra/test_utils/launcher_common_tests.py:553: error: Unsupported left operand type for / ("None")
hydra/test_utils/launcher_common_tests.py:553: note: Left operand is of type "Optional[Path]"

```

Reply:
mypy is smart enough to know if the value cannot be None.
in this case you are calling a function which may return something else the second time.
put it into a temp variable and mypy will know it can't be None.

Reply:
thx! fixed.